### ISSUE TITLE
Github OAuth Failing

### ISSUE THREAD

**Issue by connoro7** [oslabs-beta/QueryIQ#8]
# Github OAuth Failing

User sign in is throwing an error when trying to sign in with Github. I am able to navigate to the sign in page at `http://localhost:3000/api/auth/signin?callbackUrl=http%3A%2F%2Flocalhost%3A3000%2F`, and when I click on Sign in with Github, it prints the following to the server console:

Logs:
```
API handler should not return a value, received object.
prisma:query SELECT `main`.`Account`.`id`, `main`.`Account`.`userId` FROM `main`.`Account` WHERE ((`main`.`Account`.`provider` = ? AND `main`.`Account`.`providerAccountId` = ?) AND 1=1) LIMIT ? OFFSET ?
prisma:error 
Invalid `p.account.findUnique()` invocation in
/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45

  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),
  209 async getUserByAccount(provider_providerAccountId) {
  210     var _a;
→ 211     const account = await p.account.findUnique(
The table `main.Account` does not exist in the current database.
[next-auth][error][adapter_error_getUserByAccount] 
https://next-auth.js.org/errors#adapter_error_getuserbyaccount 
Invalid `p.account.findUnique()` invocation in
/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45

  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),
  209 async getUserByAccount(provider_providerAccountId) {
  210     var _a;
→ 211     const account = await p.account.findUnique(
The table `main.Account` does not exist in the current database. {
  message: '\n' +
    'Invalid `p.account.findUnique()` invocation in\n' +
    '/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45\n' +
    '\n' +
    '  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),\n' +
    '  209 async getUserByAccount(provider_providerAccountId) {\n' +
    '  210     var _a;\n' +
    '→ 211     const account = await p.account.findUnique(\n' +
    'The table `main.Account` does not exist in the current database.',
  stack: 'Error: \n' +
    'Invalid `p.account.findUnique()` invocation in\n' +
    '/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45\n' +
    '\n' +
    '  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),\n' +
    '  209 async getUserByAccount(provider_providerAccountId) {\n' +
    '  210     var _a;\n' +
    '→ 211     const account = await p.account.findUnique(\n' +
    'The table `main.Account` does not exist in the current database.\n' +
    '    at An.handleRequestError (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6929)\n' +
    '    at An.handleAndLogRequestError (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6358)\n' +
    '    at An.request (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6237)\n' +
    '    at async getUserByAccount (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:29)',
  name: 'Error'
}
[next-auth][error][OAUTH_CALLBACK_HANDLER_ERROR] 
https://next-auth.js.org/errors#oauth_callback_handler_error 
Invalid `p.account.findUnique()` invocation in
/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45

  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),
  209 async getUserByAccount(provider_providerAccountId) {
  210     var _a;
→ 211     const account = await p.account.findUnique(
The table `main.Account` does not exist in the current database. Error: 
Invalid `p.account.findUnique()` invocation in
/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:45

  208 getUserByEmail: (email) => p.user.findUnique({ where: { email } }),
  209 async getUserByAccount(provider_providerAccountId) {
  210     var _a;
→ 211     const account = await p.account.findUnique(
The table `main.Account` does not exist in the current database.
    at An.handleRequestError (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6929)
    at An.handleAndLogRequestError (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6358)
    at An.request (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@prisma/client/runtime/library.js:174:6237)
    at async getUserByAccount (/Users/connor/__Dev/codesmith/osp/QueryIQ/node_modules/@next-auth/prisma-adapter/dist/index.js:211:29) {
  name: 'GetUserByAccountError',
  code: 'P2021'
}
API handler should not return a value, received object.
API handler should not return a value, received object.

```

**Comment by connoro7** [comment]
```js
@see `~/prisma/schema.prisma` // Account schema
```

**Comment by connoro7** [comment]
Resolved #10
